src/buildstream_plugins/elements/autotools.yaml

```diff
@@ -31,9 +31,6 @@ variables:
   # Element-specific extra arguments to be passed to `configure`.
   conf-local: ''
 
-  # For backwards compatibility only, do not use.
-  conf-extra: ''
-
   conf-cmd: "%{conf-root}/configure"
   
   conf-args: |
```

Comment:
maybe we could take the opportunity to stop using `|` and use `>` instead? (and drop the backslashes)

Reply:
Why would that be more desirable ?

Reply:
Also wouldn't that make all of the configure args wrap horribly on one line when printed in the UI rather than being nicely formatted into multiple lines with `\` line breaks fed to the shell ?

Reply:
Why would that be horrible?

Reply:
Because it would not be clearly readable and comprehensive in the UI ?

I think that’s pretty self evident.

Reply:
No, that's your personal opinion. We would prefer it changed as users of BuildStream. But it's not that big a deal IMHO even though it's ugly like it is.

Reply:
I wouldn't have put it the way @nanonyme did, but I agree that "horrible" is a matter of personal opinion.

I'm fine with it either way, although I have a slight preference for not using backslashes.

Reply:
I’d be happy if the backslashes went away, but they are required to split lines in shell scripts.

I’m sure that if we thought it were more legible to read all on a single line wrapping around the terminal, then we would also write it all in one line in the YAML file.

Reply:
It is perfectly fine terminal wraps the output.

Reply:
Reading the build log and observing precisely how a module is configured will be always be important (for example when wondering why your system doesn’t behave quite how you expected it to), so having the logs be as human readable as possible is very important.

The subjective prettiness of the YAML file itself cannot be as important as this, and anyway, it will never be edited again, so there is no reason to look at it.

Except when displaying the defaults in documentation, where again, it is a good idea to show good practice of using the backslashes, because they actual have practical value in rendering more clear and readable logs.

Reply:
Well, the opposite seems to typically be true for readability, hence why we typically don't downstream use | for args. But this is anyhow subjective.